lib/Logger.ts

```diff
@@ -4,48 +4,36 @@ import winston from 'winston';
 
 import utils from './utils/utils';
 
+enum Levels {
```

Comment:
Let's stick to singular TS enum naming (`Level`)

Reply:
Good call, just forgot to rename these.